content/docs/inline-editing/inline-blocks.md

```diff
@@ -58,6 +58,32 @@ interface BlockComponentProps {
 
 Since it **renders in 'edit mode,'** this component should display `BlocksControls` and at least one _Inline Field_. `BlocksControls` is the UI for editing, deleting, or moving blocks. An _Inline Field_ is, at its most basic, an input field stripped of styling to blend in with the site.
 
+#### Blocks Controls Options
+
+| Key             | Description                                                                                                                                                                                                                                                                                                                        |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`          | The path to some value in the data being edited. If no value is provided, the child fields will reference the root of the source file.                                                                                                                                                                                             |
```

Comment:
`BlocksControls` doesn't have a `name` prop, it accesses it via the `InlineFieldContext`
```suggestion
```

Reply:
good catch, thanks